packages/terra-slider/src/Slider.module.scss

```diff
@@ -0,0 +1,212 @@
+@import '~terra-mixins/lib/Mixins';
+
+// Themes
+@import './orion-fusion-theme/Slider.module';
+@import './clinical-lowlight-theme/Slider.module';
+
+:local {
+  .slider {
+    align-items: center;
+    display: inline-flex;
+    min-height: 44px;
+  }
+
+  .label {
+    color: var(--terra-slider-label-color, #1c1f21);
+    font-weight: 700;
+    margin-right: 7px;
+    text-align: center;
+    white-space: initial;
+    width: 20%;
+  }
+
+  .min-label {
+    color: #6f7477;
+    font-size: var(--terra-slider-range-label-font-size, 12px);
+    font-weight: 400;
+    margin-right: 7px;
+    text-align: left;
+    white-space: initial;
+  }
+
+  .max-label {
+    color: #6f7477;
+    font-size: var(--terra-slider-range-label-font-size, 12px);
+    font-weight: 400;
+    margin-left: 7px;
+    text-align: left;
+    white-space: initial;
+  }
```

Comment:
You have the same values for both classes.  You should just create one class for both the min and max labels.